pep-0665.rst

```diff
@@ -1,79 +1,97 @@
 PEP: 665
-Title: Specifying Installation Requirements for Python Projects
+Title: A file format to list Python dependencies for reproducibility of an application
 Author: Brett Cannon <brett@python.org>,
         Pradyun Gedam <pradyunsg@gmail.com>,
         Tzu-ping Chung <uranusjr@gmail.com>
-PEP-Delegate:
-Discussions-To: https://discuss.python.org/t/pep-665-specifying-installation-requirements-for-python-projects/9911
+PEP-Delegate: Paul Moore <p.f.moore@gmail.com>
+Discussions-To: https://discuss.python.org/t/9911
 Status: Draft
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 29-Jul-2021
-Post-History: 29-Jul-2021
+Post-History: 29-Jul-2021, 03-Nov-2021
 Resolution:
 
 ========
 Abstract
 ========
 
-This PEP specifies a file format to list the Python package
-installation requirements for a project. The list of projects is
-considered exhaustive for the installation target and thus
-*locked down*, not requiring any information beyond the platform being
-installed for and the *lock file* listing the required dependencies
-to perform a successful installation of dependencies.
+This PEP specifies a file format to specify the list of Python package
+installation requirements for an application, and the relation between the
+specified requirements. The list of requirements is considered
+exhaustive for the installation target, and thus not requiring any
+information beyond the platform being installed for, and the file
+itself. The file format is flexible enough to allow installing the
+requirements across different platforms, which guarantees
+reproducibility on multiple platforms from the same file.
+
+===========
+Terminology
+===========
+
+There are several terms whose definition must be agreed upon in order
+to facilitate a discussion on the topic of this PEP.
+
+A *package* is something you install as a dependency and use via an
+``import`` statement. The packages on PyPI are an example of this.
```

Comment:
I think this should be the import system. The user might use the import statement, but equally valid to use ``__import__`` or the `importlib` package.